src/tasks/Crossgen2Tasks/ResolveReadyToRunCompilers.cs

```diff
@@ -9,6 +9,7 @@
 
 using Microsoft.Build.Framework;
```

Comment:
Good question. I'll check.

Reply:
In case crossgen2 modifications are necessary, its copy in sdk repo would need to be updated. I'm not sure why we have two copies, but the one in dotnet/sdk is definitely in use (which is a bit annoying as we need to [wait for the next preview](https://github.com/dotnet/runtime/pull/96941#issuecomment-1916860251) to get the changes working in runtime, despite this 'live' copy).

Reply:
This is not needed for NativeAOT. And more surprisingly, it also seems unneeded to publish ReadyToRun using a source-built Crossgen2 package.